datasources/src/main/scala/com/normation/plugins/datasources/QueryService.scala

```diff
@@ -244,7 +244,7 @@ class HttpQueryDataSourceService(
     for {
       mode          <- globalPolicyMode()
       updated       <- tasks(nodes, info.policyServers, mode, info.parameters).timeout(timeout).provide(clock).notOptional(
-                         s"Timout error after ${timeout.asScala.toString()}"
+                         s"Timeout error after ${timeout.asScala.render()}"
```

Comment:
so it seems that there is 12 different durations, not all with render, let me see what is happening

Reply:
Ok, it's that one: ```s"Timout error after ${zio.duration.Duration.fromJava(timeout).render}"```